app/views/photos/index.html.erb

```diff
@@ -0,0 +1,5 @@
+<% @photos.each do |photo| %>
+  <div><img src="<%= photo.image_path %>" alt=""></div>
```

Comment:
`image_path` is actually a field on their model. They're sourcing external images for now.